readme.md

```diff
@@ -0,0 +1,315 @@
+# Celestia Node gRPC API Design Decisions & Motivations
```

Comment:
super small nit: it may be good to split design decisions and overall design into a DESIGN.md file allowing the readme to be short sweet and how to get started. 

Secondly, will the developer be required to dive into node to work with this api, understand assumptions made there or can i spin up a client and submit?

Reply:
Fixed first in 8022b0b05c2b847e1311c6862121fb11d9e77fd2

Reply:
You should be able to just use the API